RSSReader/Services/RSSParser.m

```diff
@@ -0,0 +1,86 @@
+//
+//  RSSParser.m
+//  RSSReader
+//
+//  Created by Владислав Станкевич on 19.11.20.
+//
+
+#import "RSSParser.h"
+#import "FeedItem.h"
+#import "NSString+DateConverter.h"
+#import "NSXMLParser+InitializationWithDelegate.h"
+
+@interface RSSParser () <NSXMLParserDelegate>
+
+@property (nonatomic, copy) void (^completion)(NSError *);
+@property (nonatomic, retain)NSMutableArray<FeedItem *>* array;
+@property (nonatomic, retain)NSString *currentElement;
+
+@end
+
+NSString *kItem = @"item";
+NSString *kTitle = @"title";
+NSString *kDescription = @"description";
+NSString *kLink = @"link";
+NSString *kPubDate = @"pubDate";
```

Comment:
if you suppose use them as constants, use `NSString * const <name>`

Reply:
fixed